src/map/README.md

```diff
@@ -297,11 +305,39 @@ Function. TODO
 
 ## `onClick`
 
-Function. TODO
+Function. Returns an event:
+```
+    {
+        latitude,
+        longitude,
+        isCluster:bool,
+        count:int - will be -1 if cluster size can't be determined
+        features: [
+            {
+                layer: {
+                    id:string,
+                    group:string - matches mapbox:group defined in style.json
+                },
+                properties:[] - all available properties,
+                title:string - human readable title,
+                fields: [ - human readable properties
+                    {
+                        id:string,
+                        label:string,
+                        value:any,
+                        title:string,
+                        isLink:bool
+                    }
+                ]
+            }
+        ],
+        feature: {} - set if there's only one feature
```

Comment:
We have duplicated feature, is this something legacy to maintain and keep compatibility? if not I'd remove it to keep only the array of features

Reply:
You're right: https://github.com/GlobalFishingWatch/map-components/pull/57/commits/30d3b018692f9ef22483bb7d5ee110a444dcca28#diff-69b34ee7f05dfc863ca8b45572c8827bR335